env.example

```diff
@@ -45,15 +45,4 @@ TELEGRAM_CHAT_ID=-0000000000000
 ###############################################################################
 
 USE_IMGUR_FOR_IMAGES=true
-IMGUR_CLIENT_ID=0000000000
-
-
-###############################################################################
-#                                                                             #
-#                      Misc. required env variables                           #
-#                -- not meant for user config, don't touch --                 #
-#                                                                             #
-###############################################################################
-
-# https://github.com/yagop/node-telegram-bot-api/issues/319#issuecomment-324963294
-NTBA_FIX_319=1
+IMGUR_CLIENT_ID=0000000000
```

Comment:
I left a note elsewhere, but please change this default to `7d6b00b87043f58`, as this is what we are going to use as TeleIRC's default Imgur token.